apps/nerves_hub_www/lib/nerves_hub_www_web/templates/audit_log/_audit_log_feed.html.eex

```diff
@@ -7,9 +7,9 @@
         <div class="audit-action-icon icon-<%= audit_log.action %>"></div>
         <div>
           <p class="audit-description"><%= audit_log.description %></p>
-          <div class="help-text">
-            <%= DateTimeFormat.from_now(audit_log.inserted_at) %>
-          </div>
+            <div class="help-text">
+              <%= if !is_nil(audit_log.inserted_at) do DateTimeFormat.from_now(audit_log.inserted_at) end %>
```

Comment:
I think we can remove this check because `inserted_at` is a required field at the DB level.